package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "bcrypt-nodejs": "0.0.3",
     "body-parser": "^1.17.1",
+    "colored-console":"^0.10.0",
```

Comment:
Can you please use a package that is more popular and better maintained? Like https://www.npmjs.com/package/colors seems to be a better choice here.

Reply:
I tried colors again, and it didn't work in Webstorm terminal see https://github.com/Marak/colors.js/issues/196, another issue is it doesn't display Object member for better output.  Attached is comparison display with colors on the right. I have the changes in stashed if you still want to use colors

![colored-console](https://user-images.githubusercontent.com/4011094/30306461-804fb386-97a2-11e7-9543-79e6895e1573.jpg)

Reply:
I think you could use the logger we already have at https://github.com/Sirimangalo/meditation-plus-server/blob/master/src/app/helper/logger.js . You can use `logger.warn` for red and `logger.info` for green (more here https://github.com/winstonjs/winston#using-logging-levels).